miqi/sandbox/manager.py

```
@@ -59,33 +60,190 @@ def sandbox_is_active(sandbox_manager: Any) -> bool:
     )
 
 
-def describe_exec_environment(sandbox_manager: Any) -> str:
+_git_bash_checked = False
+_git_bash_path: str | None = None
+
+
+def _is_cygwin_bash(path: str) -> bool:
+    """True when *path* is a Cygwin bash.exe (e.g. C:\cygwin64\bin\bash.exe).
```

Ruff (0.16.1)

[warning] Invalid escape sequence: `\c`

Add backslash to escape sequence

(W605)